CONTRIBUTING.md

```diff
@@ -0,0 +1,37 @@
+# Before you contribute to the SPDX License List
+Please familiarize yourself with the SPDX License List and its supporting documentation, so that you understand the pertinent context around the list itself:
+* [Overview of the SPDX License List](https://spdx.org/spdx-license-list/license-list-overview) explains some key background, principles for the inclusion of a license on the SPDX License List, as well as an explanation of the fields contained on the list.
+* [SPDX License List Matching Guidelines](https://spdx.org/spdx-license-list/matching-guidelines) provides guidelines to be used for the purposes of matching licenses and license exceptions against those included on the SPDX License List. 
+
+# How to contribute/participate: general
+The SPDX License List is maintained by the SPDX Legal Team. Work for the SPDX Legal Team is done both via the mailing list and bi-weekly calls. In particular, the bi-weekly calls are used to discuss topics and issues that may be difficult to only discuss via email.
+You can join the mailing list and otherwise manage your subscription [via spdx-legal mailman](https://lists.spdx.org/mailman/listinfo/spdx-legal). Information on the bi-weekly calls and the working area for the SPDX Legal Team, can be found on the [SPDX wiki](https://wiki.spdx.org/view/Legal_Team)
+
+## Request a new license or exception be added to the SPDX License List
+The SPDX Legal Team greatly appreciates proposals for new open source licenses or exceptions to be added to the SPDX License List.  To request that a license or exception be added to the SPDX License List, please:
+
+1)  First, review the [license list overview and inclusion principles](https://spdx.org/spdx-license-list/license-list-overview).
+
+2)  Check the historical license and exceptions [tracking page](https://docs.google.com/spreadsheets/d/11AKxLBoN_VXM32OmDTk2hKeYExKzsnPjAVM7rLstQ8s/edit?pli=1#gid=695212681) and Issues with label ADD to ensure this license or exception has not been previously requested. 
```

Comment:
I'd rather drop the historical tracking page entirely.  Is there a lot of information there that isn't represented in issues/PRs for this repository?  If so, I think we should do a one-off data dump into new issues (e.g. for licenses/exceptions which were proposed but subsequently rejected).  That's a one-time cost for the translator (I can volunteer if this sounds like a useful direction), and it saves hopefully many contributors time by giving them a single place to look.

Reply:
that is a good idea. I think we'd want to still save the wiki page (and just mark is as archived, same for spreadsheet) and then have a very concise, consistent format for the old ones. I imagine these as added as "new" issues but then closed as the licenses/exception were either added or rejected. We might also use more specific labels reflecting that as well.

Reply:
> …then have a very concise, consistent format for the old ones…

Do you want to have a go at one to define that format, and then I can go through and follow that example with the rest?